layers/+lang/windows-scripts/packages.el

```diff
@@ -53,6 +53,8 @@
         (spacemacs|define-text-object "$" "dollarparen" "$(" ")")
         )
       (add-hook 'powershell-mode-hook 'powershell/define-text-objects)
+      ;; set powershell indent to tab-width
+      (setq powershell-indent tab-width)
```

Comment:
Doesn't it matter that `tab-width` is buffer-local? There's `(default-value 'tab-width)` if using the default value (set by `setq-default`) makes more sense here. (I don't actually know what's appropriate here, just raising a concern)